src/Microsoft.DotNet.Darc/src/Darc/Operations/SubscriptionStatusOperation.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Darc.Helpers;
+using Microsoft.DotNet.Darc.Models.PopUps;
+using Microsoft.DotNet.Darc.Options;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.Maestro.Client.Models;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Darc.Operations
+{
+    class SubscriptionStatus : Operation
+    {
+        SubscriptionStatusCommandLineOptions _options;
+
+        public SubscriptionStatus(SubscriptionStatusCommandLineOptions options)
+            : base(options)
+        {
+            _options = options;
+        }
+
+        /// <summary>
+        /// Implements the 'subscription-status' operation
+        /// </summary>
+        /// <param name="options"></param>
+        public override async Task<int> ExecuteAsync()
+        {
+            if ((_options.Enable && _options.Disable) ||
+                (!_options.Enable && !_options.Disable))
+            {
+                Logger.LogError("'enable' and 'disable' options should have different values...");
```

Comment:
Use Console.WriteLine here rather than LogError, since it generates hard to read messages.
I'd change this to say:
'Please specify either --enable or --disable'